spec/factories/advanced_setting.rb

```diff
@@ -0,0 +1,4 @@
+FactoryBot.define do
+  factory :ibm_power_hmc_advanced_setting,
+          :class => "AdvancedSetting"
+end
```

Comment:
@vfebvre it is a little confusing that this is called `ibm_power_hmc_advanced_setting` when it is actually just the base class.  Since you aren't setting anything in the factory FactoryBot can "figure it out" from the name and define the factory on the fly.  Try just deleting this file, and using just `FactoryBot.create(:advanced_settings, :name => "pcm_enabled", :resource => host, :value => "false")` in your host_spec

Reply:
Ok, fixed it. Interesting, I did not know factories could be defined implicitly.